src/Aspire.Hosting.Azure.Provisioning/Provisioners/AzureCosmosDBProvisioner.cs

```diff
@@ -76,5 +93,36 @@ public override bool ConfigureResource(IConfiguration configuration, AzureCosmos
 
         var connectionStrings = context.UserSecrets.Prop("ConnectionStrings");
         connectionStrings[resource.Name] = resource.ConnectionString;
+
+        foreach (var database in resource.Databases)
+        {
+            await CreateDatabaseIfNotExists(cosmosResource, database, cancellationToken).ConfigureAwait(false);
+            connectionStrings[database.Name] = resource.ConnectionString; // Same as parent resource.
+        }
+    }
```

Comment:
You're not handling deleting unused ones. Take a look at what service bus does for queues and topics.

Reply:
Done.